tests/integration_tests/test_wet_run_errors.py

```diff
@@ -38,3 +39,13 @@ def test_wet_run_errors(
         assert search_caplog_contains(
             caplog, "command is:", "--ignore fieldmaps slicetiming"
         )
+
+        meta_file = Path.cwd() / "output/.metadata.json"
+        assert meta_file.exists()
+        with open(meta_file) as json_file:
+            metadata = json.load(json_file)
+            assert (
+                "Elapsed (wall clock) time (h:mm:ss or m:ss)"
+                in metadata["analysis"]["info"]["resources used"]
+            )
+            assert len(metadata["analysis"]["info"]["resources used"]) == 23
```

Comment:
Will this ever change, based on system?

Reply:
This is GNU time, specifically installed because it does more than the shell time built-in.  So it shouldn't change for any system unless GNU time changes.  I guess we'll find out.